### PR TITLE
DM-44770: allow for not making campaign ancillary files and / or tagging input …

### PR DESCRIPTION
**Pull request by eacharles**
…data

### CODE REVIEW

File: examples/example_HSC_DRP-Prod_start.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: MUST_OVERRIDE
+parent_name: HSC_DRP-Prod
+spec_name: HSC_DRP-Prod_htcondor
+collection:
+    out: MUST_OVERRIDE
+#   campaign_source: HSC/raw/PDR2/WIDE,HSC/raw/PDR2/DEEP,HSC/raw/PDR2/UDEEP
+data:
+    prod_area: MUST_OVERRIDE
+    lsst_version: MUST_OVERRIDE
+#   butler_repo: '/repo/main+sasquatch_dev'
```
Comment (fritzm) [inline examples/example_HSC_DRP-Prod_start.yaml:10]: If these commented line represent potential things that can still be added here, okay, but if they are old/outdated things, please remove?

File: examples/example_HSC_DRP-RC2_start.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name: MUST_OVERRIDE
+parent_name: HSC_DRP-RC2
+spec_name: HSC_DRP-RC2_htcondor
+collection:
+    out: MUST_OVERRIDE
+#   campaign_source: HSC/raw/RC2
+data:
+    prod_area: MUST_OVERRIDE
+    lsst_version: MUST_OVERRIDE
+#   butler_repo: '/repo/main+sasquatch_dev'
```
Comment (fritzm) [inline examples/example_HSC_DRP-RC2_start.yaml:10]: same per above

File: examples/example_HSC_DRP-RC2_subset_start.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+name: MUST_OVERRIDE
+parent_name: HSC_DRP-RC2_subset
+spec_name: HSC_DRP-RC2_subset_htcondor
+collection:
+    out: MUST_OVERRIDE
+#   campaign_input: HSC/RC2_subset/defaults
+data:
+    prod_area: MUST_OVERRIDE
+    lsst_version: MUST_OVERRIDE
+#   butler_repo: '/repo/main'
```
Comment (fritzm) [inline examples/example_HSC_DRP-RC2_subset_start.yaml:10]: same per above

File: examples/example_LATISS_DRP_start.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+name: MUST_OVERRIDE
+parent_name: LATISS_DRP
+spec_name: LATISS_DRP_htcondor
+collection:
+    out: MUST_OVERRIDE
+#   campaign_input: LATISS/defaults
+data:
+    prod_area: MUST_OVERRIDE
+    lsst_version: MUST_OVERRIDE
+#   butler_repo: '/repo/embargo'
```
Comment (fritzm) [inline examples/example_LATISS_DRP_start.yaml:10]: ...beginning to think the comments here might be left on purpose...

File: examples/test_script.py [hunk 5]
```diff
@@ -1,29 +1,44 @@
...
-        # os.system(com2)
 
-        # os.system("sleep 30")
+for wms in ["panda", "htcondor"]:
+    for key, val in prod_dict.items():
+        com2 = f"cm-client action process --fullname {key}/{test_name}_{wms} --fake_status accepted"
+        print(com2)
+        if not dry_run:
+            pass
+            # os.system(com2)
```
Comment (fritzm) [inline examples/test_script.py:43]: same q: retain or remove these comments?

**Review by fritzm — APPROVED** [review]
LGTM -- just some habitual questions re. commented yaml and code.  Good to remove dead wood if it is really dead, so it doesn't pile up (only you can prevent forest fires!)

**Comment by eacharles** [comment]
The comments are there so that people can see what the defaults are for things they might want to override.

File: examples/example_HSC_DRP-Prod_start.yaml [hunk 6]
```diff
@@ -0,0 +1,22 @@
+name: MUST_OVERRIDE
+parent_name: HSC_DRP-Prod
+spec_name: HSC_DRP-Prod_htcondor
+collection:
```
Comment (yalsayyad) [inline examples/example_HSC_DRP-Prod_start.yaml:4]: As you noticed in pair coding, `collection` is spelled `collections`. Other than that, looks OK.

**Pull request merged**